.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,41 @@
+name: Release snapshot
+
+# Releases a snapshot release when new commits merge to develop
+# This ensures the release process is working as expected as well as always gives us a release
+# we should move to a single branch in near future
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
> Releases a snapshot release when new commits merge to **develop**

Are we releasing a snapshot on commits to develop or the main branch?

Reply:
fixed